### PR TITLE
Update connector-testpanel.adoc

### PR DESCRIPTION
**Pull request by george-hoffer**

### CODE REVIEW

File: modules/ROOT/pages/connector-testpanel.adoc [hunk 1]
```diff
@@ -2,20 +2,22 @@
...
-:keywords: hl7, testpanel
 
-TestPanel is an HL7 receiver, transmitter, and message editor. TestPanel is free software distributed under the MPL/GPL licenses, and runs on Mac OSX, Linux, and Windows.
+https://hapifhir.github.io/hapi-hl7v2/hapi-testpanel/[HAPI TestPanel] is an HL7 receiver, transmitter, and message editor. TestPanel is free software distributed under the http://hl7api.sourceforge.net/license.html[MPL/GPL] licenses, and runs on Mac OSX, Linux, and Windows. 
+
+Before using this document, download, install, and start TestPanel using the https://hapifhir.github.io/hapi-hl7v2/hapi-testpanel/install.html[HAPI TestPanel site]. Thereafter, you can use the information in this document to send HL7 messages to the TestPanel and receive messages from the software.
 
 Because the TestPanel can transmit HL7 messages to a listening application through MLLP or HTTP, you can use it to test your Mule flows that handle HL7. The following section provides an example of a simple flow tested with the TestPanel.
 
+*Sample Application:* https://docs.mulesoft.com/downloads/healthcare-toolkit/3.0/hl7-edi-demo-app.zip[HL7 EDI MLLP Demo Application (zip file)]
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/connector-testpanel.adoc]: @george-hoffer Let's not use bold to highlight things--it's a signal to translation that this is UI and it won't be translated. The link will highlight it enough?

File: modules/ROOT/pages/connector-testpanel.adoc [hunk 2]
```diff
@@ -41,59 +43,59 @@ image::testflow-diag.png[testflow_diag]
 This example application performs the following actions:
 
 * Listens for incoming HL7 messages.
-* After it receives a message through HL7 MLLP, HL7 EDI reads and validates a message, and generates an HL7 ACK message.
+* After it receives a message through HL7 MLLP, HL7 EDI reads/validates a message and generates an HL7 ACK message.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/connector-testpanel.adoc]: Let's not use slash as it's difficult to translate.

**Review by Mulesoft-Mysti — APPROVED** [review]
I made two comments related to translation.

File: modules/ROOT/pages/connector-testpanel.adoc (hunk 1 above)
Comment (george-hoffer) [reply modules/ROOT/pages/connector-testpanel.adoc]: Fixed.

File: modules/ROOT/pages/connector-testpanel.adoc (hunk 2 above)
Comment (george-hoffer) [reply modules/ROOT/pages/connector-testpanel.adoc]: Fixed.

**Pull request merged**